specification/machinelearningservices/resource-manager/readme.md

```diff
@@ -24,11 +24,6 @@ To see additional help and options, run:
 
 These are the global settings for the Machine Learning Services API.
 
-``` yaml
-openapi-type: arm
-tag: package-2020-08-01
-```
```

Comment:
Please add these lines back and the error `no title specified` reported in SDK generation would be resolved